app/uk/gov/nationalarchives/omega/editorial/models/package.scala

```diff
@@ -42,11 +42,24 @@ package object editSetRecords {
         |  "endDateYear" : "1962",
         |  "legalStatus": "ref.1",
         |  "placeOfDeposit" : "1",
-        |  "note": "A note about COAL.2022.V1RJW.P."
+        |  "note": "A note about COAL.2022.V1RJW.P.",
+        |  "relatedMaterial" : [
+        |    {
```

Comment:
An empty object causes a RTE.

Reply:
As in, an object like:
```
        |  "relatedMaterial" : [
        |    {
        |    },
...
```
?
The runtime exception is caused by the JSON object failing to parse as related material. I think that behaviour is expected?

Reply:
A safe assumption that it will have either a description or a link.

Of course, if the class consisted of two optional fields that this would pass.